example/App.tsx

```diff
@@ -131,8 +132,16 @@ const EXAMPLES: ExamplesSection[] = [
       { name: 'Bouncing box', component: BouncingBox },
       { name: 'Pan responder', component: PanResponder },
       { name: 'Horizontal drawer', component: HorizontalDrawer },
-      { name: 'Pager & drawer', component: PagerAndDrawer },
-      { name: 'Force touch', component: ForceTouch },
+      {
+        name: 'Pager & drawer',
+        component: PagerAndDrawer,
+        unsupportedPlatforms: new Set(['web', 'ios', 'macos']),
+      },
+      {
+        name: 'Force touch',
+        component: ForceTouch,
+        unsupportedPlatforms: new Set(['web', 'android']),
```

Comment:
Does force touch work on macOS?

Reply:
Theoretically it could. In practice, we get warning that force touch is not available. I think we can disable this example, right?

Reply:
I've done it in [9fcc315](https://github.com/software-mansion/react-native-gesture-handler/pull/3135/commits/9fcc31595e5e1092f9d1fb0482bc1e7b3dc32cee)